README.md

```diff
@@ -49,6 +49,25 @@ Bond also provides `with-stub!`. It works the same as `with-spy`, but redefines
     
 ```
 
+Private functions can also be stubbed or spyed:
```

Comment:
Private functions can't be `spy`ed, but there's a very small change to enable it, I'll add that to this PR.